common/djangoapps/microsite_configuration/backends/database.py

```diff
@@ -0,0 +1,202 @@
+"""
+Microsite backend that reads the configuration from the database
+
+"""
+from mako.template import Template
+from util.cache import cache
+
+from django.dispatch import receiver
+from django.db.models.signals import post_save
+
+from microsite_configuration.backends.filebased import SettingsFileMicrositeBackend
+from microsite_configuration.backends.base import BaseMicrositeTemplateBackend
+from microsite_configuration.models import (
+    Microsite,
+    MicrositeOrgMapping,
+    MicrositeTemplate
+)
+from microsite_configuration.microsite import get_value as microsite_get_value
+
+
+class DatabaseMicrositeBackend(SettingsFileMicrositeBackend):
```

Comment:
Seems odd to extend SettingsFileMicrositeBackend here -- maybe we should instead have a BaseMicrositeBackend, and then DatabaseMicrositeBackend and SettingsFileMicrositeBackend can extend from that?